packages/experimental-nextjs-app-support/src/ssr/index.ts

```diff
@@ -1,35 +1,17 @@
-export { ApolloNextAppProvider } from "../ApolloNextAppProvider.js";
-export { resetManualSSRApolloSingletons as resetNextSSRApolloSingletons } from "@apollo/client-react-streaming/manual-transport";
-import { ApolloClient } from "@apollo/client-react-streaming";
-import { bundle } from "../bundleInfo.js";
 export {
   InMemoryCache as NextSSRInMemoryCache,
+  ApolloClient as NextSSRApolloClient,
```

Comment:
Do we need a way to mark these as deprecated? Or is there another way we can hint that these exports will be removed in a future version?

Reply:
I was thinking about that, too.... I think I'll merge this as-is and add another PR for that.